CMakeLists.txt

```diff
@@ -293,6 +293,20 @@ if(WITH_LIBUSB)
   endif(LIBUSB_1_FOUND)
 endif(WITH_LIBUSB)
 
+# realsense2
+option(WITH_REALSENSE2 "Build with realsense 2.0 (librealsense) support " TRUE)
+if(WITH_REALSENSE2)
+  find_package(realsense2)
+
+  if(realsense2_LIBRARY)
+    # remap realsense2 cmake vars so they are compatible with the PCL_ADD_GRABBER_DEPENDENCY macro
+    set(REALSENSE2_FOUND TRUE)
```

Comment:
```patch
- if(realsense2_LIBRARY)
-   set(REALSENSE2_FOUND TRUE)
+ if(realsense2_FOUND)
+   set(REALSENSE2_FOUND ${realsense2_FOUND})
```

Reply:
I used realsense2_LIBRARY (instead of realsense2_FOUND) in the condition here because i don't think the librealsense cmake files set any sort of FOUND flag - but I might have missed it..

Reply:
(I don't know where it is defined, But) It seems that it is defined.
```cmake
find_package(realsense2)
message(STATUS "${realsense2_FOUND}") # output 1
```

Reply:
I'll fix this Monday, thanks for testing!

Reply:
fixed